python/lsst/daf/persistence/butlerFactory.py

```diff
@@ -26,7 +26,7 @@
 
```

Comment:
Do we still need this class at all?  I think all the places that use it can be converted to just the `Butler()` constructor.